scripts/gen-hsclient.js

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const assert = require('bsert');
+const path = require('path');
+const os = require('os');
+const util = require('util');
+const cp = require('child_process');
+const exec = util.promisify(cp.exec);
+const fs = require('bfile');
+const Config = require('bcfg');
+
+const ROOT = path.dirname(__dirname);
+const NAME = 'hs-client';
+const HSD_PKG = require(path.join(ROOT, 'package.json'));
+
+const REMOTE = 'git@github.com:handshake-org/hs-client.git';
+
+const INIT_HS_CLIENT_PKG = {
+  name: 'hs-client',
+  description: 'HSD node and wallet client',
+  keywords: [
+    'http',
+    'request',
+    'socket.io',
+    'websockets'
+  ],
+  main: './lib/client/index.js',
+  bin: {
+    'hsd-cli': './bin/hsd-cli',
+    'hsd-rpc': './bin/hsd-rpc',
+    'hsw-cli': './bin/hsw-cli',
+    'hsw-rpc': './bin/hsw-rpc'
+  },
+  engines: {
+    node: '>=8.0.0'
```

Comment:
does this need to be documented somewhere in a release process or something to keep track and update?
I see hsd's package says `>=14.0.0` and this one is lower probably because it's just a client. Anyway, just more of an fyi